sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -264,7 +265,9 @@ def test_lasso_cv():
     )
     # check that they also give a similar MSE
     mse_lars = interpolate.interp1d(lars.cv_alphas_, lars.mse_path_.T)
-    assert_allclose(mse_lars(clf.alphas_[5]).mean(), clf.mse_path_[5].mean(), rtol=1e-2)
+    np.testing.assert_approx_equal(
```

Comment:
We rather use `assert_allclose` and set `rtol` and `atol` instead of specifying the tolerance using a number of significant decimals.